CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 5.0.0-76
+Feature: Update bookNav shortcut menu ordering
```

Comment:
Let's please follow the pattern here. Bullet, include the person, and it looks like this is more of a fix than a feature.